client/hotrod-client/src/test/java/org/infinispan/client/hotrod/query/projection/RemoteDistributedMetaProjectionTest.java

```diff
@@ -0,0 +1,35 @@
+package org.infinispan.client.hotrod.query.projection;
+
+import static org.infinispan.configuration.cache.IndexStorage.LOCAL_HEAP;
+import static org.infinispan.server.hotrod.test.HotRodTestingUtil.hotRodCacheConfiguration;
+
+import org.infinispan.client.hotrod.test.MultiHotRodServersTest;
+import org.infinispan.configuration.cache.CacheMode;
+import org.infinispan.configuration.cache.ConfigurationBuilder;
+import org.infinispan.protostream.SerializationContextInitializer;
+import org.infinispan.query.model.Developer;
+import org.testng.annotations.Test;
+
+@Test(groups = "functional", testName = "org.infinispan.client.hotrod.query.projection.RemoteDistributedMetaProjectionTest")
+public class RemoteDistributedMetaProjectionTest extends MultiHotRodServersTest {
+
+   @Override
+   protected void createCacheManagers() throws Throwable {
+      ConfigurationBuilder config = hotRodCacheConfiguration(getDefaultClusteredCacheConfig(CacheMode.DIST_SYNC));
+      config.indexing().enable()
+            .storage(LOCAL_HEAP)
+            .addIndexedEntity("io.dev.Developer");
+      createHotRodServers(8, config);
```

Comment:
do you need 8 servers?

Reply:
Just to be sure about the distribution of the entries among different nodes. We can reduce it to 2 though.

Reply:
You can set the num-owners=1 to make each node have a disjoint key set.